wp-content/themes/currentorg/inc/shortcode-current-ltw-projects.php

```diff
@@ -86,13 +86,58 @@ function current_ltw_projects_assets() {
 	wp_register_script(
 		'current-ltw-script',
 		get_stylesheet_directory_uri() . '/js/current-ltw-projects.js',
-		array(),
+		array( 'jquery' ),
 		filemtime( get_stylesheet_directory() . '/js/current-ltw-projects.js' ),
 		true
 	);
+	wp_localize_script( 
+		'current-ltw-script', 
+		'ajax_object',
+		array( 'ajax_url' => admin_url( 'admin-ajax.php' ) )
+	);
 }
 add_action( 'wp_enqueue_scripts', 'current_ltw_projects_assets' );
 
+/**
+ * Load a single project when it is selected from the project list
+ */
+function current_ltw_projects_load_single_project_callback() {
+
+	// make sure post id is set and actually a valid post
+    if( isset( $_GET['post_id'] ) && get_post_status( $_GET['post_id'] ) ) {
+
+		$post_id = $_GET["post_id"];
+		
+		// set up basic args for our query to grab the post
+		$single_project_args = array(
+			'post_type' => 'projects',
+			'p' => $post_id
```

Comment:
This needs to be sanitized; since post IDs are always integers we could do `abs(intval( $post_id ))`

Reply:
Updated in https://github.com/INN/umbrella-currentorg/pull/140/commits/554eded8f38056c544c42325d34e873084ce3fdb